main.py

```diff
@@ -1,10 +1,37 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        json_data = json.load(file)
+    for name in json_data:
+        race, created = Race.objects.get_or_create(
```

Comment:
You could omit the `created` argument with `_` here and on line 22 because it is not used in the logic. Apply this change.